### PR TITLE
Implement Linear+BatchNorm1d fusion

### PR DESCRIPTION
**Pull request by dreiss**
Summary: Only handles eval mode for now.

Test Plan: Unit test

Reviewed By: raghuramank100

Differential Revision: D21282486

### CODE REVIEW

File: torch/quantization/fuse_modules.py [hunk 1]
```diff
@@ -32,6 +33,27 @@ def fuse_conv_bn(conv, bn):
     else:
         return torch.nn.utils.fuse_conv_bn_eval(conv, bn)
 
+def fuse_linear_bn(linear, bn):
```
Comment (vadimkantorov) [inline torch/quantization/fuse_modules.py:36]: There is a naming inconsistency wrt `fuse_conv_bn_eval` which has the suffix `eval`. Is there a reason for this naming divergence?

File: torch/nn/utils/fusion.py [hunk 2]
```diff
@@ -23,3 +23,23 @@ def fuse_conv_bn_weights(conv_w, conv_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
...
+
+    fused_linear.weight, fused_linear.bias = fuse_linear_bn_weights(
+        fused_linear.weight, fused_linear.bias,
+        bn.running_mean, bn.running_var, bn.eps, bn.weight, bn.bias)
+
+    return fused_linear
+
+def fuse_linear_bn_weights(linear_w, linear_b, bn_rm, bn_rv, bn_eps, bn_w, bn_b):
+    if linear_b is None:
+        linear_b = bn_rm.new_zeros(bn_rm.shape)
```
Comment (vadimkantorov) [inline torch/nn/utils/fusion.py:39]: should this be `linear_b = torch.zeros_like(bn_rm)`?
